vagrant/provisioning/roles/showcase/templates/startup.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
```

Comment:
When we remove the Angular resource copier from ArkCase, then the ArkCase deployment won't require yarn, node, or npm, and we would remove those tasks from the ArkCase installer.  When that happens, we can add them to this showcase installer... but should be good for now, if we assume showcase is always installed with ArkCase.